GameModes/OnlineGameMode.cs

```diff
@@ -149,5 +165,47 @@ public virtual void LobbyReadyCheck()
         { 
             
         }
+
+        internal virtual void PlayerLeftLobby(OnlinePlayer player)
+        {
+            
+        }
+
+        internal virtual void NewPlayerInLobby(OnlinePlayer player)
+        {
+
+        }
+
+        internal virtual void LobbyTick(uint tick)
+        {
+            
+        }
+
+        internal virtual void Customize(Creature creature, OnlineCreature oc)
+        {
+            if (lobby.playerAvatars.Any(a=>a == oc.id))
+            {
+                RainMeadow.Debug($"Customizing avatar {creature} for {oc.owner}");
+                var settings = lobby.entities.Values.First(em => em.entity is AvatarSettings avs && avs.target == oc.id).entity as AvatarSettings;
+
+                // this adds the entry in the CWT
+                var mcc = RainMeadow.creatureCustomizations.GetValue(creature, (c) => settings.MakeCustomization());
+
+                if(creature is Player player && !oc.isMine)
+                {
+                    player.controller = new OnlineController(oc, player);
+                }
+
+                // todo one day come back to making emote support universal
```

Comment:
I was going to ask about this but it's not an expectation at the moment with story players so perhaps for another time.